p4-16/psa/PSA.mdk

```diff
@@ -1149,17 +1151,25 @@ values that should be associated with packets cloned using that
 session.
 
 ```
-PortId_t         egress_port;
+/// Each clone session may configure one or more pairs of (egress_port, instance).
+PortId_t         egress_port;  /// egress_port in a pair of (egress_port, instance)
+EgressInstance_t instance;  /// instance in a pair of (egress_port, instance)
+
+/// Each clone session may configure at most one of each following values.
 ClassOfService_t class_of_service;
 bool             truncate;
 PacketLength_t   packet_length_bytes;  /// only used if truncate is true
 ```
 
 The `egress_port` may be any port that can be used for normal unicast
-packets, i.e. any normal port, `PSA_PORT_CPU`, or `PSA_PORT_RECIRCULATE`.  For
-the latter two values, the cloned packet will be sent to the CPU, or
-recirculated at the end of egress processing, as a normal unicast
-packet would at the end of egress processing.
+packets, i.e. any normal port, `PSA_PORT_CPU`, or
```

Comment:
Have we not defined 'normal port' before? If not, maybe we should, and then we can use it throughout.